client/entrypoints/payment-request-settings/payment-request-section.js

```diff
@@ -8,18 +8,16 @@ import interpolateComponents from 'interpolate-components';
 import { Elements } from '@stripe/react-stripe-js';
```

Comment:
`useState` can be removed as an import, no longer being used.

Reply:
Ah thanks! Done in 865df282